roles/taiga/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
     that:
       - "taiga_secret_key is defined"
       - "taiga_rabbitmq_password is defined or not (taiga_enable_events | bool or taiga_enable_async_tasks | bool)"
-      - "taiga_service_manager == 'circus' or ansible_version | version_compare('2.2', '>=')"
+      - "taiga_service_manager == 'circus' or ansible_version is version_compare('2.2', '>=')"
```

Comment:
This version comparison is only in there because Ansible versions prior to 2.2 did not support the `systemd` module. The README has stated for a while that Ansible 2.5 is the minimal supported version, so this test is really kind of useless. :) Thus I think that instead of modifying it, we can actually drop this line altogether.

Reply:
Noted, Here we go 👍

Reply:
As mentioned, that line can go completely, rather than just the Ansible version check.

Fixed in #14 now, thanks again for spotting this!